src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp

```diff
@@ -775,6 +775,33 @@ class npc_salanar_the_horseman : public CreatureScript
     }
 };
 
+enum HorseSeats
+{
+    SEAT_ID_0                       = 0
+};
+
+class spell_stable_master_repo : public AuraScript
+{
+    PrepareAuraScript(spell_stable_master_repo);
+
+    void AfterApply(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+    {
+        Creature* creature = GetTarget()->ToCreature();
+        if (!creature)
+            return;
+
+        if (Unit* passenger = creature->GetVehicleKit()->GetPassenger(SEAT_ID_0))
```

Comment:
Please add a null check on the vehicle kit to

Reply:
Done.